Documentation/devel/quickstart-dev.md

```diff
@@ -0,0 +1,92 @@
+# Preparing development environment and first rkt build
+This is an example configuration for a fresh installation on Ubuntu 16.04 GNOME.
+This doc is also a short quick start guide.
+For a detailed reference see [hacking.md](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md)
+### Get rkt repo and install dependencies
+In this example ~/Repos is a personal workspace where all repos are stored
+```sh
+$ mkdir ~/Repos && cd ~/Repos
+$ mkdir -p ~/.local/gopath/src/github.com/coreos
+$ sudo apt-get install git
+$ git -C ~/.local/gopath/src/github.com/coreos clone https://github.com/coreos/rkt.git
+$ ln -s ~/.local/gopath/src/github.com/coreos/rkt rkt
+```
+On a fresh system installation, few additional software packages are needed to correctly build rkt:
+```sh
+$ sudo ~/Repos/rkt/scripts/install-deps-debian-sid.sh
+```
+See also the dependencies [page](https://github.com/coreos/rkt/blob/master/Documentation/dependencies.md).
+### Installing Go Programming Language for a single-user
```

Comment:
Each heading needs a blank line before and after it.
This should be a level two heading (`##`).